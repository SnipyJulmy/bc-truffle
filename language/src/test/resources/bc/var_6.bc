b = 12
b
define id(a) {
  a
  return a
}
define f(b) {
  auto a
  a = b
  a
  return b
}
define g(b) {
  auto a
  a = b
  a
  return a
}
id(12)
f(13)
g(14)
id(12)
f(13)
g(14)
id(56)
f(213124)
g(937382)
halt
