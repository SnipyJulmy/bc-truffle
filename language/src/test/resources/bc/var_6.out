12
12
12
13
13
14
14
12
12
13
13
14
14
56
56
213124
213124
937382
937382
